sdk/personalizer/azure-ai-personalizer/src/test/java/com/azure/ai/personalizer/ConfigurationTests.java

```diff
@@ -0,0 +1,82 @@
+package com.azure.ai.personalizer;
+
+import com.azure.ai.personalizer.implementation.models.PolicyContract;
+import com.azure.ai.personalizer.implementation.models.ServiceConfiguration;
+import com.azure.core.http.HttpClient;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.time.Duration;
+
+import static com.azure.ai.personalizer.TestUtils.DISPLAY_NAME_WITH_ARGUMENTS;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class ConfigurationTests extends PersonalizerTestBase {
```

Comment:
Should this be called `PersonalizerClientTest` ?

Reply:
Called it ConfigurationTests since it tests out the configuration apis.